jdaviz/core/template_mixin.py

```diff
@@ -3965,6 +3965,8 @@ def __init__(self, plugin, value, default, auto,
                          default=default, auto=auto,
                          invalid_msg=invalid_msg)
 
+        if getattr(plugin, auto):
+            setattr(plugin, value, getattr(plugin, default))
```

Comment:
this ensures that the `value` will be updated to `default` on init (wasn't previously necessary because of the timing of when default was first assigned in all other uses of `AutoTextField`.

Reply:
Should your comment be added to code inline comment so devs will know in the future?

Reply:
good idea - added.  I don't think the context of why its added now is needed in the code, but useful to give some context since the code might be a little obfuscated otherwise.